bisection.py

```diff
@@ -459,14 +459,15 @@ def run(self):
                 targets = left.digest.keys()
             if targets == None and len(right.digest):
                 targets = right.digest.keys()
-            updated_targets = self.regression(left, right, targets)
-            if len(updated_targets):
-                mid = self.torch_src.get_mid_commit(left, right)
-                if mid == None:
-                    self.result.append((left, right))
-                else:
-                    self.bisectq.append((left, mid, updated_targets))
-                    self.bisectq.append((mid, right, updated_targets))
+            # Commented out for now.
```

Comment:
is this because bisection.py isn't easy to comment out entirely in some other script?  I guess we won't benefit from bisection at all until we start supporting the regular torchbench timing scripts.

Reply:
That's the case. I just want to reuse the building process of bisection.py.